backends-common/rabbitmq/pom.xml

```diff
@@ -93,6 +93,16 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-pool2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.13</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.5</version>
```

Comment:
I can see we have an httpcore artifact declared in `james-server-jmap` module as well, in a different version (4.4.14). Would be good to try centralizing the use of the artifact in the root pom and bump it for jmap